internal/pkg/agent/configuration/fleet.go

```diff
@@ -47,5 +54,40 @@ func DefaultFleetAgentConfig() *FleetAgentConfig {
 		Enabled: false,
 		Client:  remote.DefaultClientConfig(),
 		Info:    &AgentInfo{},
+		Checkin: DefaultFleetCheckin(),
+	}
+}
+
+func DefaultFleetCheckin() FleetCheckin {
+	return FleetCheckin{
+		Mode:               fleetCheckinModeStandard,
+		RequestBackoffInit: 60 * time.Second,
+		RequestBackoffMax:  10 * time.Minute,
```

Comment:
This is defined twice, once here and once in fleet_gateway.go and it looks like this one takes precedence because it's non-zero. If someone changes the settings in fleet_gateway.go they'll be ignored as far as I can tell which is confusing. Can you define these in a single place?

https://github.com/elastic/elastic-agent/blob/dc21d56dbf15d948a395f3c2370339450cb2bcb0/internal/pkg/agent/application/gateway/fleet/fleet_gateway.go#L43-L47

Reply:
Yeah sure I don't have a preference which one is the only one, as long as it isn't defined in two places that don't interact in an obvious way.